Automattic/wp-calypso
# Site Profiler: Add the Health and Security sections

Fixes https://github.com/Automattic/dotcom-forge/issues/7305
Fixes https://github.com/Automattic/dotcom-forge/issues/7306

## Proposed Changes

* Add the health and security sections
* The security section will not be shown the site is not WordPress

## Why are these changes being made?

To match the [figma layout](https://www.figma.com/design/P7Jjk4nbUTahUk8aHkG6NQ/Performance-Tool?m=dev&node-id=1126-6311) for Site Profiler V2.

## Testing Instructions

* Go to `/site-profiler/:url` with a WP.com site. Ex: `/site-profiler/wordpress.com`
* Go to the Health and Security sections to check if they are properly populated
* Go to `/site-profiler/:url` with a WP.com site. Ex: `/site-profiler/example.com`
* Go to the Health and Security sections to check if they are properly populated


## Pre-merge Checklist
